### ISSUE TITLE
Bug in copy button of context popup menu 

### ISSUE THREAD

**Issue by NishantWankhade** [UDAAN-LEAP/leap-pe-tool#298]
How to get the bug :- 
- Select some text
- Press right click and press copy from popup menu
You will notice that, the word is not reflected to the clipboard of the context popup menu.

**Comment by NishantWankhade** [comment]
Have done some progress at commit 74b23e66c87218d0813bf3f2448122409d24ff38 

However there is a **bug** in it , .... 
If you go to copy same word may number of times it shows that many times on the clipboard... the expected behavior should be one word should come once in the clipboard.

**Comment by NishantWankhade** [comment]
@saanvibehele @AayushGupta69 Please test this feature at your system as well.

**Comment by AayushGupta69** [comment]
@NishantWankhade Yes, can confirm that bug. Have started working on a fix for that.
